src/main/java/dk/cngroup/lentils/service/OrganizerService.java

```diff
@@ -55,7 +55,7 @@ public User getOrganizerById(final Long id) {
 
     public List<OrganizerFormDTO> getOrganizerDtos() {
         return getOrganizers().stream()
-                .map(organizer -> organizerMapper.map(organizer))
+                .map(organizerMapper::map)
```

Comment:
toto je tvuj fix (vypada to na nejaky rebase problem)

Reply:
jojo, spatny rebase